Documentation/install/upgrade.rst

```diff
@@ -279,6 +241,16 @@ latest ``1.1.y`` release before subsequently upgrading to ``1.2.z``.
 +-----------------------+-----------------------+-----------------------+-------------------------+---------------------------+
 | ``1.5.x``             | ``1.6.y``             | Required              | Minimal to None         | Clients must reconnect[1] |
 +-----------------------+-----------------------+-----------------------+-------------------------+---------------------------+
+| ``1.6.x``             | ``1.6.6``             | Not required          | Minimal to None         | Clients must reconnect[1] |
++-----------------------+-----------------------+-----------------------+-------------------------+---------------------------+
+| ``1.6.x``             | ``1.6.7``             | Required              | Minimal to None         | Clients must reconnect[1] |
++-----------------------+-----------------------+-----------------------+-------------------------+---------------------------+
+| ``1.6.x``             | ``1.7.y``             | Required              | Minimal to None         | Clients must reconnect[1] |
++-----------------------+-----------------------+-----------------------+-------------------------+---------------------------+
+| ``1.7.0``             | ``1.7.1``             | Required              | Minimal to None         | Clients must reconnect[1] |
++-----------------------+-----------------------+-----------------------+-------------------------+---------------------------+
```

Comment:
Maybe we should switch the column name "``DaemonSet`` upgrade" to something else just to give the impression that the k8s descriptor needs to be re-generated.